frontend/src/v5/ui/v4Adapter/overrides/preview/previewComments.overrides.ts

```diff
@@ -72,6 +72,10 @@ export const CommentListStyling = css`
 	${CommentListContainer} {
 		padding: 0;
 
+		${EmptyStateInfo} {
+			margin: 2px 0 0;
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/8986c943-d565-4b95-9785-bc871b3f0d14)
Remove button is now positioned weirdly

Reply:
I don't think that bit of code is responsible for it, the problem seems to be in staging already